css/card.css

```diff
@@ -284,10 +290,6 @@ section label {
   margin-bottom:0.5em;
 }
 
-#search-bar {
```

Comment:
This was causing the entire section to be bumped down, so removed it here and added a media query instead to apply the same for smaller screens.